content/dev/dev-guide.md

```diff
@@ -0,0 +1,264 @@
+ - Let's create a new `hello` desk.
+ - First navigate to the `pkg` directory in `urbit-git`
+   - ```bash
+     cd ~/urbit-git/pkg
+     ```
+ - Next create a `hello` directory
+   - ```bash
+     mkdir hello
+     ```
+ - Next run the included `symbolic-merge.sh` script from the `pkg` directory.
+   - ```bash
+     ./symbolic-merge.sh base-dev hello
+     ./symbolic-merge.sh garden-dev hello
+     ```
+ - Now we should see `lib`, `mar`, and `sur` inside `hello`.
+ - Our desk must include a `sys.kelvin` file which specifies the kernel version it's compatible with. Let's create that.
+   - ```bash
+     echo "[%zuse 420]" > sys.kelvin
+     ```
+ - We can also add a `desk.ship` file to specify the original publisher of this desk. Since we're going to use our fake ship `~zod` let's add `~zod` as the publisher.
+   - ```bash
+     echo "~zod" > desk.ship
+     ```
+ - If we had Gall agents in this desk which should be automatically started when the desk is installed, we'd add them to a hoon list in the `desk.bill` file. It would look something like this:
+   - ```bash
+      :~  %some-app
+          %another
+      ==
+     ```
+   - Since we're not adding any agents in this example, we'll **omit the `desk.bill` file**.
+ - The final file we need is `desk.docket-0`. This one's more complicated, so we'll need to edit it in a text editor.
+   - ```bash
+     nano desk.docket-0
+     ```
+   - In the text editor paste the following and save:
+   - ```bash
+     :~  title+'Hello'
+         info+'A simple hello world app.'
+         color+0x81.88c9
+         image+'https://media.urbit.org/docs/userspace/dist/wut.svg'
+         base+'hello'
+         glob-ames+[~zod 0v0]
+         version+[0 0 1]
+         website+'https://urbit.org/docs/userspace/dist/guide'
+         license+'MIT'
+     ==
+     ```
+  - This is a `docket` file. This file sets various options for the desk with a tile and usually a brower-based front-end of some kind. 
+  - It mainly configures the appearance of an app's tile, the source of its glob (bundle of client resources like HTML, JS, and CSS files), and some additional metadata.
+    - We've given the app a `%title` of "Hello", which will be displayed on the app tile and will be the name of the app when others browse to install it. We've given the app tile a `%color` of #8188C9, and also specified the URL of an `%image` to display on the tile.
+    - The `%base` clause specifies the base URL path for the app. We've specified "hello" so it'll be `http://localhost:80/apps/hello/...` in the browser. 
+    - For the glob, we've used a clause of `%glob-ames`, which means the glob will be served from a ship over Ames, as opposed to being served over HTTP with a `%glob-http` clause or having an Eyre binding with a `%site clause`. 
```

Comment:
Also the `:80` is dependent on urbit being able to bind to that. It might already be in use by something else, for example. Probably worth pointing out the in-use port can be found among the startup messages.